apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx

```diff
@@ -372,7 +379,9 @@ export const TOTPAuthenticator: React.FunctionComponent<TOTPProps> = (
     const initDeleteTOTP = (): void => {
         deleteTOTP().then(() => {
             setIsTOTPConfigured(false);
-            enabledAuthenticatorsUpdateHandler(EnabledAuthenticatorUpdateAction.DELETE_TOTP);
+            if (enableMFAUserWise === true && isSuperTenantLogin == true) {
```

Comment:
Just having `if (enableMFAUserWise` will be true for truth values.
```suggestion
            if (enableMFAUserWise && isSuperTenantLogin) {
```